docs/framework/data/adonet/ef/aggregate-functions-sqlclient-for-entity-framework.md

```diff
@@ -61,12 +59,11 @@ An `Int32`.
 
 **Example**
 
-[!code-csharp[DP EntityServices Concepts#SQLSERVER_COUNT](~/samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entitysql.cs#sqlserver_count)]
```

Comment:
The link in line 65 is broken. (I have no idea why.)

Reply:
Interesting. I decided to copy from a working one and apparently it was missing a closing ].